python/imjoy_rpc/connection/colab_connection.py

```diff
@@ -1,13 +1,12 @@
-import base64
+"""Provide a colab connection."""
 import contextvars
 import logging
 import os
 import sys
 import uuid
 
-import google
```

Comment:
google package is not used anymore so it's good to remove it. However, I am thinking maybe we should insert a test later, in the start function to make sure we are actually running in colab. Maybe try to import google and if it fails we blow it up?